### ISSUE TITLE
Unused variable 'kMinExp'

### ISSUE THREAD

**Issue by zerodefect** [nlohmann/json#935]
**Bug Report**

- What is the issue you have?

Unused variable 'kMinExp' in to_chars.hpp, specifically in function:

```
template <typename FloatType>
boundaries compute_boundaries(FloatType value)
``` 

then see:

```
constexpr int      kMinExp    = 1 - kBias;
```

Depending on level of compiler warnings/errors, this can fail a build.

- Please describe the steps to reproduce the issue. Can you provide a small but working code example?

Using cmake with ExternalProject_add() and building code with '-Wunused-variable' and '-Werror'

- What is the expected behavior?

Successful build.

- And what is the actual behavior instead?

Compiler error.

- Which compiler and operating system are you using? Is it a [supported compiler](https://github.com/nlohmann/json#supported-compilers)?

Clang 4.0

- Did you use a released version of the library or the version from the `develop` branch?

Released, but it looks to be a problem in the 'develop'

- If you experience a compilation error: can you [compile and run the unit tests](https://github.com/nlohmann/json#execute-unit-tests)?

Yes.

**Comment by abolz** [comment]
Could you open a PR and replace `1-kBias` in line 207 with `kMinExp`?

**Comment by zerodefect** [comment]
I'll do that. I wasn't sure if there was a code bug in there, so I thought I'd raise the problem before making changes.

**Comment by nlohmann** [comment]
Fixed by merging  #936.
